.github/workflows/test.yml

```diff
@@ -128,6 +136,59 @@ jobs:
           rm -rf .kernel/.git
           cp -rf .kernel/. .
           rm -rf .kernel
+      - if: ${{ github.repository != 'kernel-patches/vmtest' }}
```

Comment:
Good question. It mostly just has not been done. It's a big time sink to get this kind of logic tested on a single repository, because of the incredibly slow iteration times (keep in mind that for testing during normal Pacific Time business hours the CI is typically as busy as it gets, causing runs to take up to several hours). Two repositories certainly don't help.

But, I argue, it's also not that important there: pull request are much less frequent and further spaced apart (in time). That means that the base line has more time to diverge, which naturally causes more compilation. It may also result in caches being evicted by the time a new pull request is created. That, combined with the fact that there are just so few pull requests to begin with, means that there is much less to gain there.

I think we should look into it, if for no other reason than to get these two or so lines removed, but I wouldn't treat it as a priority at this point. Let me know if you disagree.

Reply:
Actually...there is an actual hard reason, now that I think about it more, and that is that we don't have a baseline in the `vmtest` repository. Basically, because of the hoops we have to jump through due to restrictions by `actions/cache`, we require the `bpf_base` and `bpf-next_base` branches to be present and kept up-to-date by KPD (I didn't require that initially, but it's the only way I could make everything work). That won't happen in `vmtest` (and I don't foresee that changing). We could presumably get somewhere near that with regularly scheduled builds (which I understand is a GitHub Actions feature), but that will require a bit more work. I will make this more clear in the description/code.